packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -84,6 +85,7 @@ function persistSceneConfig(
 		valueDB.setMetadata(sceneIdValueId, {
 			...ValueMetadata.UInt8,
 			label: `Associated Scene ID (${groupId})`,
+			valueChangeOptions: ["transitionDuration"],
```

Comment:
I think that this is insufficient, because nodes that have already been interviewed prior to this change will not get updated Metadata. What else should be done for this?

Reply:
ATM we can't do much else than to tell users to re-interview.